core/events/events_click.js

```diff
@@ -33,10 +34,6 @@ Blockly.Events.Click = function(opt_block, opt_workspaceId, opt_targetType) {
   Blockly.Events.Click.superClass_.constructor.call(this, workspaceId);
   this.blockId = opt_block ? opt_block.id : null;
 
-  if (!opt_targetType && !this.isBlank) {
-    opt_targetType = opt_block ? 'block' : 'workspace';
-  }
-
```

Comment:
At first the only 2 types were 'block' and 'workspace', but now it also used for zoom control ui click and it seems confusing to have it try to do something smart rather than explicitly passing it. Removing this seems like a better idea to me so there's no confusion of whether you need to pass the targetType.